Kp update

kqp5476 opened the pull request:
**Summary of changes:** 
Reordered top right buttons (info, comments, and home); 
Added corresponding URLs to top right buttons; 
Left sidebar menu (added references page); 
Included all appropriate info on References page; 
Revised "Acknowledgements" on Overview page; 
More specific text on buttons on Overview and Prereq pages; 
Matched layout on Challenge page to that of style guide; 
Improved readability of calculation output on Challenge page; 
Inserted boastUtils to standardize fonts, colors, copyright info, etc.

kqp5476 committed:
standardized formatting in menuItem functions; removed hyphen from prerequisites, changed name button on prerequ page

kqp5476 committed:
made code explicit, changed column layout to sidebar layout

kqp5476 committed:
still not working
:(

kqp5476 committed:
what needs to be done:
change order of top right buttons;
swap well panels on top of challenge page

kqp5476 committed:
aligned math text on challenge page :D

kqp5476 committed:
changed order of top right buttons; switched well panels on Challenge page

shravanisamala commented on app.R:
```diff
@@ -0,0 +1,541 @@
+                textOutput("question"),
+                br(), 
+                bsButton(
+                  inputId = "show_ans", 
```
Might need to add style, class, and color too

shravanisamala replied:
Also I think you need to add a Description file because the style guide doesn't have CODEOWNERS anymore!

shravanisamala commented on ui.R:
```diff
@@ -10,117 +10,151 @@ shinyUI(fluidPage(
-                                )))
-                              )
-                            )
-                            )))
```
Github says there should be a newline here

shravanisamala commented on ui.R:
```diff
@@ -10,117 +10,151 @@ shinyUI(fluidPage(
+              )
+      )
+    ))
+))
```
Github says there should be a newline here

shravanisamala reviewed (dismissed):
Left some comments in the code!

neilhatfield committed:
Repo Restructure

I removed several unneeded files and move some files to better reflect the current structure of app repos.

neilhatfield commented:
@kqp5476 Is there a DESCRIPTION file on this branch that I'm just not seeing?

kqp5476 committed:
Create DESCRIPTION

neilhatfield committed:
Merge pull request #8 from EducationShinyAppTeam/kqp5476-patch-1

Created DESCRIPTION file

rpc5102 committed:
Fixed: DESCRIPTION

neilhatfield committed:
initial set of updates

neilhatfield committed:
Updates to plot

Created new method for generating sample data, moved plot into ggplot2. Old code is still present for the time being. Also changed variable names to be more meaningful

neilhatfield committed:
Swapping Code

Finished swapping out old code for new code. Now waiting for input from epidemiologist

rpc5102 committed:
rLocker additions

rpc5102 committed:
Hide answer on new question.

neilhatfield committed:
Update app.R

Additional layout updates based on feedback

neilhatfield committed:
Merge pull request #9 from EducationShinyAppTeam/fall2021Updates

Fall2021 updates
